nat-lab/tests/utils/vm/windows_vm_util.py

```diff
@@ -39,6 +39,22 @@ async def new_connection() -> AsyncIterator[Connection]:
         await _copy_binaries(ssh_connection, connection)
         await _disable_firewall(connection)
 
+        async def on_stdout(stdout: str) -> None:
+            print(
+                "********************\n",
+                stdout,
+                "********************",
+            )
+
+        connection.create_process(["route", "print"]).run(
+            stdout_callback=on_stdout,
+            stderr_callback=on_stdout,
+        )
+        connection.create_process(["ipconfig/all"]).run(
+            stdout_callback=on_stdout,
+            stderr_callback=on_stdout,
+        )
+
```

Comment:
It looks like some debug, do you really want to merge it to master?

Reply:
Yes, this is a temporary merge